tests/integration/api-gateway/gateway-tests/manifests/k8s/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ory-hydra-login-consent
+  namespace: kyma-system
+spec:
+  selector:
+    matchLabels:
+      app: ory-hydra-login-consent
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: ory-hydra-login-consent
+        version: v1
+    spec:
+      containers:
+        - name: login-consent
+          image: piotrkpc/hydra-login-consent:test_0.1
```

Comment:
We can't use such image. We have to provide something more "official". Perhaps a little project in kyma-incubator.

Reply:
Currently working on it. Here is the image for testing that pipelines work.